src/supportedLocales.json

```diff
@@ -0,0 +1,18 @@
+[
+  "nl-BE",
+  "nl-NL",
+  "fr-BE",
+  "fr-FR",
+  "en-GB",
+  "de-DE",
+  "es-ES",
+  "it-IT",
+  "da-DK",
+  "pt-PT",
+  "pl-PL",
+  "tr-TR",
+  "nb-NO",
+  "fi-FI",
+  "sv-SE",
+  "tlh-KL"
+]
```

Comment:
Are we missing Catalan here? `ca-ES` ?

Reply:
Aparently we don't support Catalan in the applications as of yet, though this might change soon.